src/properties.jl

```diff
@@ -89,22 +91,56 @@ end
 
 struct SphericalVector{L, LEN, T} <: AbstractProperty
    val::SVector{LEN, T}
-   _valL::Val{L}
+   _valL::Val{L} ## Why do we need Val?
```

Comment:
probably we don't. I'm happy to explore this later. Can we keep it for now please?

Reply:
We do need this as discussed. I just forgot to remove the comment, sorry..